### PR TITLE
Actions: Add Message and AT: WIP when PR accidently tries to merge to master

### PR DESCRIPTION
**Pull request by csiefer2**
Issue: #11891
Tests: Done on private @trilinos/muelu  repo.
Stakeholder feedback: See MueSlack

NOTE:  If you're the @trilinos-autotester it does not try to get in your way.  So this shouldn't block the weekly dev->master merge.

@sebrowne

### CODE REVIEW

**Review by jhux2 — APPROVED** [review]
boom!

File: .github/workflows/flag_prs_to_master.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Flag PRs to master by adding the AT WIP keyword 
+# Unless you are the autotester, of course
+
+on:
+  pull_request:
+    types: [opened]
+    branches: [main]
```
Comment (sebrowne) [inline .github/workflows/flag_prs_to_master.yml:7]: This is the part that IDs that the PR is headed for master?

**Review by sebrowne — APPROVED** [review]
I think this looks fine, and seems like a good idea to me.

File: .github/workflows/flag_prs_to_master.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Add label
+      uses: actions-ecosystem/action-add-labels@v1     
+      if: "!contains(github.event.sender.login,'trilinos-autotester')"
+      with:
+        labels: "AT: WIP"     
+    - name: Add comment
+      uses: mshick/add-pr-comment@v2
```
Comment (sebrowne) [inline .github/workflows/flag_prs_to_master.yml:27]: What is this mshick/add-pr-comment@v2 thingy?

**Review by sebrowne — APPROVED** [review]

**Pull request merged**